libs/sdk-ui-tests-e2e/cypress/tools/widget.ts

```diff
@@ -37,6 +37,13 @@ export class Widget {
         return this;
     }
 
+    noLoading() {
```

Comment:
It should be like this `waitForWigetLoading()` to clearly

Reply:
they are not the same, waitForWigetLoading because we verify `wait` in this method. In noLoading(), we only check that there is no loading, there is no meaning if we named the method as waitForWigetNoLoading because in this method, we don't verify any `wait` 

Then when we use it, it is better to use `new Widget().noLoading()` than use `new Widget().verifyWidgetNoLoading()` or somethings like that

Reply:
Ok, It will be better when we broaden the method in case if `loading` icon exits. 
Ex:  `new Widget().isLoading(true)` or `new Widget().isLoading(false)`

Reply:
please check again, `waitChartLoaded`, `waitKpiLoaded`, and `waitTableLoaded`, these 3 methods are covered `isLoading(true)`, only isLoading(false) is not covered, so only `noLoading()`  is enough,
if we use `isLoading(expect: boolean)` as you said, `isLoading(true)` will be redundant

Reply:
Yep, but it's a different meaning. as you mention, we don't verify `wait`, we check there is no loading